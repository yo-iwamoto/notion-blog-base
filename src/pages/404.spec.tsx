import { render } from '@/__tests__/util';
import Page from './404.page';

describe('[page] /404', () => {
  it('renders /404 unchanged', async () => {
    const { container } = render(<Page />);

    expect(container).toMatchSnapshot();
  });
});
