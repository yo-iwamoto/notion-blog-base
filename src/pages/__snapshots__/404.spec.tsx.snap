// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[page] /404 renders /404 unchanged 1`] = `
<div>
  <div
    class="font-sans text-gray-900"
  >
    <div
      class="mx-4 mt-12"
    >
      <div
        class="mx-auto w-full max-w-2xl"
      >
        <main>
          <div
            class="mt-20 text-center"
          >
            <h1
              class="font-bold"
            >
              RESOURCE NOT FOUND
            </h1>
            <div
              class="h-12"
            />
            <span
              class="flex cursor-pointer items-center justify-center"
            >
              <span>
                TOP
              </span>
              <span
                class="w-2"
              />
              <svg
                class="text-gray-500"
                fill="currentColor"
                height="12"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 448 512"
                width="12"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M190.5 66.9l22.2-22.2c9.4-9.4 24.6-9.4 33.9 0L441 239c9.4 9.4 9.4 24.6 0 33.9L246.6 467.3c-9.4 9.4-24.6 9.4-33.9 0l-22.2-22.2c-9.5-9.5-9.3-25 .4-34.3L311.4 296H24c-13.3 0-24-10.7-24-24v-32c0-13.3 10.7-24 24-24h287.4L190.9 101.2c-9.8-9.3-10-24.8-.4-34.3z"
                />
              </svg>
            </span>
          </div>
        </main>
      </div>
    </div>
  </div>
</div>
`;
